Offline/OfflinePlayer/objc/OfflinePlayer/VideosViewController.m

```diff
@@ -0,0 +1,1370 @@
+                                                                        options:options
+                                                                   controlsView:controlView ];
+
+        self.playerView.frame = self.videoContainer.bounds;
+        self.playerView.autoresizingMask = (UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth);
+        self.playerView.delegate = self;
+        [self.videoContainer addSubview:self.playerView];
+    }
+}
+
+- (void)createNewPlaybackController
+{
+    if (!self.playbackController)
+    {
+        NSLog(@"Creating a new playbackController");
+
+        BCOVPlayerSDKManager *sdkManager = [BCOVPlayerSDKManager sharedManagerWithOptions:@{@"log_level": @1 }]; // level one shows logs errors to the debug console
+
+        // Publisher/application IDs not required for Dynamic Delivery
+        self.authProxy = [[BCOVFPSBrightcoveAuthProxy alloc] initWithPublisherId:nil
+                                                                   applicationId:nil];
+        
+        // You can use the same auth proxy for the offline video manager
+        // and the call to create the FairPlay session provider.
+        BCOVOfflineVideoManager.sharedManager.authProxy = self.authProxy;
+        
+        // Create the session provider chain
+        BCOVBasicSessionProviderOptions *options = [[BCOVBasicSessionProviderOptions alloc] init];
+        options.sourceSelectionPolicy = [BCOVBasicSourceSelectionPolicy sourceSelectionHLSWithScheme:kBCOVSourceURLSchemeHTTPS];
+        id<BCOVPlaybackSessionProvider> basicSessionProvider = [sdkManager createBasicSessionProviderWithOptions:options];
+        id<BCOVPlaybackSessionProvider> fairPlaySessionProvider = [sdkManager createFairPlaySessionProviderWithApplicationCertificate:nil
+                                                                                                                   authorizationProxy:self.authProxy
+                                                                                                              upstreamSessionProvider:basicSessionProvider];
+        
+        // Create the playback controller
+        id<BCOVPlaybackController> playbackController = [sdkManager createPlaybackControllerWithSessionProvider:fairPlaySessionProvider
+                                                                                                   viewStrategy:nil];
+
+        // Start playing right away (the default value for autoAdvance is NO)
+        playbackController.autoAdvance = YES;
+        playbackController.autoPlay = YES;
+
+        // Register for delegate method callbacks
+        playbackController.delegate = self;
+
+        // Retain the playback controller
+        self.playbackController = playbackController;
+
+        // Associate the playback controller with the player view
+        self.playerView.playbackController = playbackController;
+    }
+}
+
+- (void)showWarnings
```

Comment:
@timothyrodgers do you think we need either of these warnings now?
There's no FairPlay by default, and we set a default account.

Reply:
Yeah, we need the "SIMULATOR" warning, but plz add a line to the alert that you can't download videos in the simulator.

Reply:
I'll leave the account warning because one of the first things a user is likely to do with this code is to replace our test credentials with their own. Feel free to object.